### PR TITLE
[MERL-513] FaustWP

### PR DESCRIPTION
**Pull request by theodesp**

### CODE REVIEW

**Comment by TeresaGobble** [comment]
Hi Theo- there are some instances of axl in the axl-website folder; is this area of the repo meant to be renamed as well in this ticket? If not, I can take care of that when I complete MERL-522 (Stage new Faust documentation site on Atlas).

**Comment by TeresaGobble** [comment]
> Hi Theo- thank you for your hard work, this is looking great! There are some instances of `axl` in the `axl-website` folder; is this area of the repo meant to be renamed as well in this ticket? If not, I can take care of that when I complete MERL-522 (Stage new Faust documentation site on Atlas).

Ah, I see from the ticket that its scope is for the packages only, so please disregard the above.

File: packages/faustwp-core/src/server/auth/middleware.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+/**
+ * A Node handler for processing incoming requests to logout an authenticated user.
+ * This handler clears the refresh token from the cookie and returns a response.
+ *
+ * @param {IncomingMessage} req
+ * @param {ServerResponse} res
+ *
+ * @see https://faust.org/docs/next/guides/auth
```
Comment (TeresaGobble) [inline packages/faustwp-core/src/server/auth/middleware.ts:89]:
```suggestion
 * @see https://faustjs.org/docs/next/guides/auth
```

File: packages/faustwp-core/src/server/auth/middleware.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+/**
+ * A Node handler for processing incoming requests to exchange an Authorization Code
+ * for an Access Token using the WordPress API. Once the code is exchanged, this
+ * handler stores the Access Token on the cookie and redirects to the frontend.
+ *
+ * @param {IncomingMessage} req
+ * @param {ServerResponse} res
+ *
+ * @see https://faust.org/docs/next/guides/auth
```
Comment (TeresaGobble) [inline packages/faustwp-core/src/server/auth/middleware.ts:22]:
```suggestion
 * @see https://faustjs.org/docs/next/guides/auth
```

File: packages/faustwp-core/DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+# GraphQL Endpoint
+NEXT_PUBLIC_WORDPRESS_URL=http://headless.local
+```
+
+## Git Workflows
+
+We have three notable branches:
+
+- `canary` - This branch has the latest changes
+- `main` - This branch is used to deploy changes to [faust.org](https://faust.org)
```
Comment (TeresaGobble) [inline packages/faustwp-core/DEVELOPMENT.md:41]:
```suggestion
- `main` - This branch is used to deploy changes to [faustjs.org](https://faustjs.org)
```

File: packages/faustwp-core/DEVELOPMENT.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+When your feature branch includes changes to the documentation website, it's helpful to include a live preview link in the PR description. The [staging site](https://hcixzyt38dn5ak04xxcqc36lf.js.wpenginepowered.com/) is used for this purpose. You can deploy your changes to the staging site using the following steps:
+
+- Checkout and switch to the `site-dev` branch.
+- Merge your feature branch into `site-dev`.
+- Push your merge commit to `site-dev`.
+- Within about 10 minutes, the docs changes from your feature branch should be visible on the [staging site](https://hcixzyt38dn5ak04xxcqc36lf.js.wpenginepowered.com/).
+
+### Prod Site Deployment
+
+The docs on faust.org are automatically built on pushes to `main`. Updating the docs on `main` will update faust.org within 10 minutes.
```
Comment (TeresaGobble) [inline packages/faustwp-core/DEVELOPMENT.md:68]:
```suggestion
The docs on faustjs.org are automatically built on pushes to `main`. Updating the docs on `main` will update faustjs.org within 10 minutes.
```

File: packages/faustwp-core/DEVELOPMENT.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+After a successful release, a PR from `canary` to `main` is automatically created. Review and merge this PR to update faust.org.
```
Comment (TeresaGobble) [inline packages/faustwp-core/DEVELOPMENT.md:70]:
```suggestion
After a successful release, a PR from `canary` to `main` is automatically created. Review and merge this PR to update faustjs.org.
```

**Review by TeresaGobble — CHANGES_REQUESTED** [review]
Thank you for tackling this for us, Theo! I made some suggestions to revert the faust.org -> faustjs.org, since we aren't able to get the faust.org domain- everything else LGTM! 😊

**Comment by theodesp** [comment]
@TeresaGobble changes implemented you can approve now thank you!

**Review by TeresaGobble — APPROVED** [review]
LGTM, thank you Theo!

**Pull request merged**